src/lib/optimizer/strategy/dependent_group_by_reduction_rule.cpp

```diff
@@ -102,7 +106,9 @@ void DependentGroupByReductionRule::_apply_to_plan_without_subqueries(
         return LQPVisitation::VisitInputs;
       }
 
-      // Else, replace it with a ProjectionNode.
+      // Else, replace it with a ProjectionNode. For instance, SELECT DISTINCT n_name, n_regionkey FROM nation (the
+      // column order is different than in the original table) turns from [ Aggregate GROUP BY n_name, n_regionkey ] to
```

Comment:
"turns from to"? Sounds strange.

Reply:
`, which we represent by [ Aggregate GROUP BY n_name, n_regionkey ], is rewritten to ...`?

Reply:
Jop

Reply:
Hmm ... maybe I was drunk? It doesn't sound problematic to me today. Feel free to ignore. I might have misread the sentence. :-/

Reply:
🤣 
🧨 then?